docs/develop/manifest-element-ordering.md

```diff
@@ -463,10 +463,31 @@ The following sections show the manifest elements in the order in which they mus
                                     <SourceLocation>
                                     <Title>
                                     <FunctionName>
-                                    <SourceLocation>
                 <SourceLocation>
                 <Label>
                 <CommandSurface>
+        <MobileFormFactor>
+            <ExtensionPoint>
+                <Group>
+                    <Label>
+                    <Control>
+                        <Label>
+                        <Icon>
+                            <Image>
+                        <Action>
+                            <TaskpaneId>
+                            <SourceLocation>
+                            <Title>
+                            <FunctionName>
+                <Control>
+                    <Label>
+                    <Icon>
+                        <Image>
+                    <Action>
+                        <TaskpaneId>
+                        <SourceLocation>
+                        <Title>
+                        <FunctionName>
```

Comment:
```suggestion
                        <Action>
                            <SourceLocation>
                            <FunctionName>
                <Control>
                    <Label>
                    <Icon>
                        <Image>
                    <Action>
                        <SourceLocation>
                        <FunctionName>
```
To my knowledge, `<TaskpaneId>` and `<Title>` aren't supported in Outlook add-ins.